### PR TITLE
Volunteer page design feedbacks

### PR DESCRIPTION
**Pull request by hoangviet1993**
- [x] Fix missing words in "Get in touch" paragraph.
- [x] Reduce scroll time to 1 second.
- [x] Setting various breakpoint for carousel to adjust across different viewports.
I have tried really hard to make things consistent across various viewports. Here is some gif for different viewports provided by Chrome's device simulation.
Pixel 2

iPad Pro

Desktop view

### CODE REVIEW

File: static/pages/volunteer/volunteer.html [hunk 1]
```diff
@@ -300,6 +300,30 @@ <h3 class="md-display-1 volunteer-page-description-headline-padding">
...
+  }
+  @media screen and (min-width: 500px) and (max-width: 599px) {
+    .volunteer-page-mobile-profile-carousel {
+      min-height: 600px;
+    }
+    .carousel-inner {
+      height: 600px;
+    }
+  }
+  @media screen and (min-width: 600px) and (max-width: 800px){
```
Comment (seanlip) [inline static/pages/volunteer/volunteer.html]: nit: add space before {

File: static/pages/volunteer/profile_carousel_directive.html [hunk 2]
```diff
@@ -19,19 +19,19 @@
...
-        <md-card-content layout="column" class="oppia-foundation-quote-box oppia-foundation-gray-background">
-          <span class="oppia-foundation-left-align-text">&ldquo;{{slide.professionalQuote}}&rdquo;</span>
-          <cite class="oppia-foundation-cite-text-box">
-            <span class="oppia-foundation-bold-text">- {{slide.fullName}}</span>
-            <p class="oppia-foundation-no-margin"></p>
-            <span class="oppia-foundation-italic-text">{{slide.position}}</span>
-          </cite>
-        </md-card-content>
-      </div>
+        <img ng-src="{{slide.profilePictureImageFilename}}" alt="Volunteer profile image" style="width: 100%">
```
Comment (seanlip) [inline static/pages/volunteer/profile_carousel_directive.html]: All these should be deindented by 2 (i.e. left as they were), I think?

File: static/pages/volunteer/volunteer.html (hunk 1 above)
Comment (hoangviet1993) [reply static/pages/volunteer/volunteer.html]: Done!

File: static/pages/volunteer/profile_carousel_directive.html (hunk 2 above)
Comment (hoangviet1993) [reply static/pages/volunteer/profile_carousel_directive.html]: Woops! Did not mean to do this. Thanks for spotting this.

**Pull request merged**